py3status/modules/whoami.py

```diff
@@ -21,17 +19,26 @@ class Py3status:
     """
     """
     # available configuration parameters
-    cache_timeout = 1800
     format = '{username}'
 
+    class Meta:
+        deprecated = {
+            'remove': [
+                {
+                    'param': 'cache_timeout',
+                    'msg': 'obsolete parameter remove',
```

Comment:
I understand you clearly.

I would have went with `obsolete parameter // remove` myself.

I had `obsolete parameter // use 'icon_on' instead` or something earlier in one of my many changes and was asked nicely to omit that `//` part.

That string is long and may make two lines in debug. I'll add `// remove this` ?